### ISSUE TITLE
Reading optimiser.star and sampling.star from RELION produces an empty list

### ISSUE THREAD

**Issue by Guillawme** [alisterburt/starfile#5]
Hello,

First of all, thank you so much for this library, it opens up many possibilities to analyze results from RELION.

I can load `run_it???_data.star` and `run_it???_model.star` without any problem, but when trying to load certain `run_it???_optimiser.star` and `run_it???_sampling.star`, I get an empty list as a result. See `Out[4]`, `Out[6]` and `Out[8]` in the example below ran on the attached files. Is this a bug, or normal behavior? These files contain information that would be nice to be able to access programmatically.

Here is what I get with the attached example files (produced with RELION 3.1.0 from their tutorial data):

[test.zip](https://github.com/alisterburt/starfile/files/5512819/test.zip)

```
$ ls
run_it025_optimiser_2D.star  run_it025_sampling_2D.star
run_it025_optimiser_3D.star  run_it025_sampling_3D.star

$ ipython --version
7.19.0

$ ipython
Python 3.8.5 (default, Sep  4 2020, 02:22:02) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.19.0 -- An enhanced Interactive Python. Type '?' for help.
```

```python
In [1]: import starfile

In [2]: starfile.version()
Out[2]: '0.3.1'

In [3]: optimiser_2D = starfile.open('run_it025_optimiser_2D.star')

In [4]: optimiser_2D
Out[4]: []

In [5]: optimiser_3D = starfile.open('run_it025_optimiser_3D.star')

In [6]: optimiser_3D
Out[6]: []

In [7]: sampling_2D = starfile.open('run_it025_sampling_2D.star')

In [8]: sampling_2D
Out[8]: []

In [9]: sampling_3D = starfile.open('run_it025_sampling_3D.star')

In [10]: sampling_3D
Out[10]: 
[       rlnIs3DSampling  rlnIs3DTranslationalSampling  ...  rlnOffsetRangeOriginal rlnOffsetStepOriginal
 value                1                             0  ...                   4.425                  1.77
 
 [1 rows x 15 columns],
      rlnAngleRot  rlnAngleTilt
 0          45.00     80.405932
 1          56.25     70.528779
 2          33.75     70.528779
 3          45.00     60.000000
 4          67.50     60.000000
 ..           ...           ...
 187       -67.50    120.000000
 188       -45.00    120.000000
 189       -33.75    109.471221
 190       -56.25    109.471221
 191       -45.00     99.594068
 
 [192 rows x 2 columns]]
```

**Comment by alisterburt** [comment]
Hi Guillaume,

Whoops! Thanks for the bug report - I'll look into it now

**Comment by alisterburt** [comment]
Confirmed - error occurs when reading "simple" data blocks which are not followed by anything else

**Comment by alisterburt** [comment]
Fixed in version 0.3.2 - thanks and please do let me know if you find any more issues!

**Comment by Guillawme** [comment]
Wow that was fast, thank you!
